src/components/manage/Widgets/AlignWidget.jsx

```diff
@@ -50,46 +48,14 @@ const AlignWidget = (props) => {
   const intl = useIntl();
 
   const {
-    id,
-    onChange,
     actions = ['left', 'right', 'center', 'full'],
-    actionsInfoMap = {},
-    value,
+    actionsInfoMap,
   } = props;
 
-  React.useEffect(() => {
-    if (!props.value && props.default) {
-      props.onChange(props.id, props.default);
-    }
-  });
-
-  const actionsInfo = {
-    ...defaultActionsInfo({ intl }),
-    ...actionsInfoMap,
-  };
+  const actionsInfo = actionsInfoMap || defaultActionsInfo({ intl });
```

Comment:
@sneridagh  I think you should bring back 
  const actionsInfo = {
    ...defaultActionsInfo({ intl }),
    ...actionsInfoMap,
  };
otherwise the actionsInfo behaves as a total replacement and it might not be what is needed
Think about this code
 <AlignWidget
          id="align"
          title="Alignment"
          fieldSet="default"
          onChange={() => {}}
          actions={['left', 'right', 'additional']}
          actionsInfoMap={{
            additional: [imageFullSVG, 'Additional action title'],
          }}

I don't want to have to pass left and right since it's already in the defaults
but because of the condition, it's either actionsInfoMap or defaultActionsInfo

Reply:
but then I don't know how to do what you requested the other day...

Reply:
Or then you'll use the `ButtonsWidget` instead if you need it?